### PR TITLE
FIX: Allow user to quote Emojis.

### PR DESCRIPTION
**Pull request by tgxworld**
meta: https://meta.discourse.org/t/cant-copy-and-paste-emoji-and-cant-quote-emoji-cry/32725

### CODE REVIEW

File: app/assets/javascripts/discourse/lib/utilities.js [hunk 1]
```diff
@@ -97,6 +97,8 @@ Discourse.Utilities = {
     // Strip out any .click elements from the HTML before converting it to text
     var div = document.createElement('div');
     div.innerHTML = html;
+    // Find all emojis and replace with its title attribute.
+    $(div).find('img.emoji').replaceWith(function() { return this.title });
```
Comment (riking) [inline app/assets/javascripts/discourse/lib/utilities.js]: should create `var $div = $(div)` and reuse

Comment (tgxworld) [inline app/assets/javascripts/discourse/lib/utilities.js]: :+1: Thanks!

**Comment by SamSaffron** [comment]
hmmm ... side effect though is that now titles on emoji are a bit uglier :( @coding-horror  ?

**Comment by SamSaffron** [comment]
OK, this only impact alt, so I guess it is fine. :)

**Pull request merged**